docs/index.md

```diff
@@ -24,13 +24,13 @@ Welcome to the Danish Foundation Models (DFM) project, a pioneering initiative i
 
 
 
-##  Open-source models on closed-source data
+##  Open-source Development with Privacy-Focused Data Handling
 
-As many of the datasets we use either contain personally sensitive information or fall under copyright they can't be shared publicly. However, we want to share as
-much as possible from the project, while protecting privacy and adhering to copyright law.
-Thus we organize it such that all parts of the project that can be shared and those which
-can't are well-documented using datasheets and training logs.
-Furthermore, during data processing and training, the data is stored on UCloud which follows the highest standards of information security management with a [formal ISO27001 certification](https://docs.cloud.sdu.dk/intro/security.html).
+In our commitment to advancing open-source development, we strongly emphasise the ethical handling of data, particularly when it involves personally sensitive information or material under copyright. This ensures that we share as much as possible while protecting privacy and adhering to copyright laws.
```

Comment:
> adhering to copyright laws

Do we actually know that this is the case - isn't it just a grey zone? Seems a bit bold to write it like that.

Reply:
I agree. We can remove it:
> This ensures that we share as much as possible while protecting privacy ~~and adhering to copyright laws~~.